pkg/cmd/server/bootstrappolicy/constants.go

```diff
@@ -67,6 +67,7 @@ const (
 	ImagePusherRoleName       = "system:image-pusher"
 	ImageBuilderRoleName      = "system:image-builder"
 	ImagePrunerRoleName       = "system:image-pruner"
+	ImageSignerRoleName       = "system:image-signer"
```

Comment:
> How does that work when a single image is shared between several projects?

I think that `update` is the wrong verb.  It seems like signatures are an additive affair.  If liggitt signs it, deads can still sign too.

I think the get case is a little interesting.  I don't know that we want to automatically allow someone in ns-1 to see signatures created via the resource in ns-2.  The ability to do that would be good, but I haven't thought about whether it should *always* be that way.  You could end up in cases where you're seeing all sorts of signatures you know nothing about.